scripts/autotools/libmongocrypt/libmongocrypt.m4

```diff
@@ -0,0 +1,27 @@
+if test "$PHP_CLIENT_SIDE_ENCRYPTION" != "no"; then
+  AC_SUBST(MONGOCRYPT_ENABLE_TRACE, 1)
+  AC_SUBST(MONGOCRYPT_IS_POSIX, 1)
+  AC_SUBST(MONGOCRYPT_IS_WIN, 0)
+
+  PHP_MONGODB_LIBMONGOCRYPT_CFLAGS="$PHP_MONGODB_BUNDLED_CFLAGS -std=gnu99"
```

Comment:
Hopefully we can get [MONGOCRYPT-219](https://jira.mongodb.org/browse/MONGOCRYPT-219) addressed before release and remove `-std=gnu99`. Until then, can we add a TODO comment that links to that issue?